lms/static/js/student_account/views/LoginView.js

```diff
@@ -86,6 +93,10 @@
 
             resetEmail: function() {
                 this.element.hide( this.$errors );
+                this.resetMessage = this.$resetSuccess.find('.message-copy');
+                if (this.resetMessage.find('p').length == 0){
+                    this.resetMessage.append("<p>" + this.message + "</p>");
```

Comment:
This will fail the safe template linter. It needs to use ``HtmlUtils.joinHtml``.